sites/lab/pages/signin/callback/[provider].mjs

```diff
@@ -36,6 +36,7 @@ const OauthCallbackPage = ({ page, provider }) => {
       const state = urlParams.get('state')
       const code = urlParams.get('code')
       const result = await backend.oauthSignIn({ state, code, provider })
+      console.log({ state, code, result })
```

Comment:
Is this an intended change or a leftover debug statement?

Reply:
Not sure where this came from, but indeed, this file wasn't meant to be committed.